afl-system-config

```diff
@@ -54,7 +54,7 @@ if [ "$PLATFORM" = "Linux" ] ; then
   echo
   dmesg | grep -E -q 'noibrs pcid nopti' || {
     echo It is recommended to boot the kernel with lots of security off - if you are running a machine that is in a secured network - so set this:
-    echo '  /etc/default/grub:GRUB_CMDLINE_LINUX_DEFAULT="ibpb=off ibrs=off kpti=0 l1tf=off mds=off mitigations=off no_stf_barrier noibpb noibrs pcid nopti nospec_store_bypass_disable nospectre_v1 nospectre_v2 pcid=on pti=off spec_store_bypass_disable=off spectre_v2=off stf_barrier=off srbds=off noexec=off noexec32=off tsx=on tsx_async_abort=off arm64.nopauth audit=0 hardened_usercopy=off ssbd=force-off"'
+    echo '  /etc/default/grub:GRUB_CMDLINE_LINUX="ibpb=off ibrs=off kpti=0 l1tf=off mds=off mitigations=off no_stf_barrier noibpb noibrs pcid nopti nospec_store_bypass_disable nospectre_v1 nospectre_v2 pcid=on pti=off spec_store_bypass_disable=off spectre_v2=off stf_barrier=off srbds=off noexec=off noexec32=off tsx=on tsx_async_abort=off arm64.nopauth audit=0 hardened_usercopy=off ssbd=force-off"'
     echo
```

Comment:
Please reverse this.  I think GRUB_CMDLINE_LINUX_DEFAULT is the better option, it is just for (some?) cloud linux setups it does not work. or do you think different?
same for best_practices.md

Reply:
I see `GRUB_CMDLINE_LINUX_DEFAULT` as the place for good defaults that do not change the behavior of the system (e.g. quiet splash; changing the console type during boot; visual changes during boot). `GRUB_CMDLINE_LINUX` is there user adjustments go. The afl-persistent-config applies user options to improve fuzzing.

But this is just my interpretation or vague documentation. In reality the only difference is whether you want to have the settings in recovery mode or not. I would claim that recovery mode is not relevant in a fuzzing setup.

Reply:
>  In reality the only difference is whether you want to have the settings in recovery mode or not. I would claim that recovery mode is not relevant in a fuzzing setup.

I agree and that is why I proposed the change. GRUB_CMDLINE_LINUX ends up in recovery mode config, GRUB_CMDLINE_LINUX_DEFAULT does not.

Reply:
I'll revert that :) sounds good